requirements.txt

```diff
@@ -6,11 +6,11 @@ gitpython==0.3.2.1
 jinja2
 jsonschema>=2.3.0
 kombu
-mongoengine>=0.8.7,<0.9
+pymongo<3.0
+mongoengine>=0.9
```

Comment:
@m4dcoder Yeah, we should use more strict version pinning otherwise we will encounter problems again in the future -e.g. `>=0.9.1,<0.10`